app/src/main/res/layout/item_layout_favourite.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.cardview.widget.CardView
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="@dimen/dp_100"
+    android:layout_height="wrap_content"
+    android:layout_margin="@dimen/dp_10"
+    android:backgroundTint="@color/firefly">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:background="@color/firefly"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content">
+
+        <RelativeLayout
+            android:id="@+id/relativeLayoutFavorite"
+            android:layout_width="@dimen/dp_100"
+            android:layout_height="@dimen/dp_100"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent">
+
+            <com.google.android.material.card.MaterialCardView
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                app:cardCornerRadius="@dimen/dp_75"
+                app:strokeColor="@color/flamingo"
+                app:strokeWidth="@dimen/dp_5">
+
+                <ImageView
+                    android:id="@+id/imageHeroFavorite"
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:layout_alignParentTop="true"
+                    android:layout_centerHorizontal="true"
+                    android:src="@drawable/ic_launcher_foreground" />
+            </com.google.android.material.card.MaterialCardView>
+
+            <com.google.android.material.card.MaterialCardView
+                android:layout_width="@dimen/dp_40"
+                android:layout_height="@dimen/dp_40"
```

Comment:
https://stackoverflow.com/questions/37318228/how-to-make-constraintlayout-work-with-percentage-values

https://stackoverflow.com/questions/41265570/constraintlayout-aspect-ratio